FabricObserver/Observers/ObserverManager.cs

```diff
@@ -260,11 +260,15 @@ public async Task StartObserversAsync()
                             }
                         }
 
-                        // Check for new version once a day.
-                        if (!(shutdownSignaled || runAsyncToken.IsCancellationRequested) && DateTime.UtcNow.Subtract(LastVersionCheckDateTime) >= NewReleaseCheckInterval)
+                        _ = bool.TryParse(GetConfigSettingValue(ObserverConstants.CheckGithubVersion, null), out bool checkGithubVersion);

```

Comment:
because the checkGithubVersion variable is inlined (out bool...), if the TryParse fails, it will be false, which is what is expected here if the customer misspells the boolean value or leaves the setting blank, for example. This is fine.